### DISCUSSION TITLE
Implementing an Advanced driver-assistance system on airsim with manual control

### DISCUSSION THREAD

**Discussion by sameerraja93** [microsoft/AirSim#3281] [Support Q&A]
Hi,

I wanted to ask if it is possible to implement an Advanced driver-assistance system such as adaptive cruise control and lane keeping assist on airsim, if yes then can I test it by taking over the car manually with steering wheel.

**Comment by chcardoz** [comment]
Have you tried enabling api control of the car?

**Comment by suzukiresearchanddevelopment** [comment]
Yes, maybe it's possible to implement & test ADAS features like ACC, LKA within the AirSim simulation environment.
Hope it will help you. for more read this blog: [Advanced driver Assistance System components](https://www.suzukirndindia.com/blogs/what-is-advanced-driver-assistance-systems)
